requirements.txt

```diff
@@ -1,16 +1,12 @@
 configobj>=5.0.5
-cherrypy==8.9.1
+cherrypy>=3.6.0,<9.0
 ws4py>=0.3.2
-SQLAlchemy>=1.0.0
+SQLAlchemy>=1.1.0
 six>=1.5.2
 Jinja2>=2.7
 rpctools>=0.3.1
 logging_unterpolation>=0.2.0
 requests>=2.2.1
-pytest>=3.0.1
-mock>=1.0.1,<1.1
-Sphinx>=1.2.1
-coverage>=3.6
```

Comment:
Sweet, I'll take a look.  I've always seen ``test_requirements.txt`` so I've kind of assumed that was the preferred idiom, but I personally like ``requirements_test.txt`` better.

Bizarrely, https://travis-ci.org/magfest/sideboard/jobs/230546161#L241 says it's installing CherryPy, but then we get a bunch of ``ImportError: No module named 'cherrypy'`` errors in the build.  I'll take a look at what you did for the plugins and see if adapting that works any better than what I have now.